nautobot/docs/installation/http-server.md

```diff
@@ -138,16 +138,47 @@ Finally, restart the `nginx` service to use the new configuration.
 $ sudo systemctl restart nginx
 ```
 
+!!! info
+    If the restart fails, and you changed the default key location, check to make sure the `nautobot.conf` file you pasted has the updated key location. For example, CentOS 8.2.2 requires keys to be in `/etc/pki/tls/` instead of `/etc/ssl/`.
+
+## Confirm Permissions for /opt/nautobot
+
+Ensure that the `/opt/nautobot/` permissions are set to `755`.
+
+Example of correct permissions:
+```no-highlight
+[root@localhost ~]# ls -l /opt/
+total 4
+drwxr-xr-x. 11 nautobot nautobot 4096 Apr  5 11:24 nautobot
+[root@localhost ~]# 
+```
+
+If the permissions are not correct, modify them accordingly.
+
+Example of modifying the permissions:
+```no-highlight
+[nautobot@localhost ~]$ ls -l /opt/
+total 4
+drwx------. 11 nautobot nautobot 4096 Apr  5 10:00 nautobot
+[nautobot@localhost ~]$ chmod 755 /opt/nautobot/
+[nautobot@localhost ~]$ ls -l /opt/
+total 4
+drwxr-xr-x. 11 nautobot nautobot 4096 Apr  5 11:24 nautobot
+```
+
 ## Confirm Connectivity
 
-At this point, you should be able to connect to the HTTPS service at the server name or IP address you provided.
+At this point, you should be able to connect to the HTTPS service at the server name or IP address you provided. The port number is no longer necessary in the URL in the base config.
 
 !!! info
     Please keep in mind that the configurations provided here are bare minimums required to get Nautobot up and running. You may want to make adjustments to better suit your production environment.
 
 !!! warning
     Certain components of Nautobot (such as the display of rack elevation diagrams) rely on the use of embedded objects. Ensure that your HTTP server configuration does not override the `X-Frame-Options` response header set by Nautobot.
 
+!!! important
+    If you used a self-signed certificate, you will likely need to explicitly allow connectivity in your browser.
```

Comment:
I don't like having three separate callout boxes in a row here. Would it make sense to add this into the `info` block above instead of creating a new box?

Reply:
Agreed.  I moved it up and added it after the initial sentence for this section.  It's not a callout anymore.  I think that flows better.  LMK.

```
At this point, you should be able to connect to the HTTPS service at the server name or IP address you provided. If you used a self-signed certificate, you will likely need to explicitly allow connectivity in your browser.
```